src/pages/Onboard/NameTeam.vue

```diff
@@ -165,6 +257,29 @@ export default {
           <v-col v-if="revealNote" key="name" cols="12" class="pb-0">
             <div class="display-1 text-center">
               Let's start by creating your team
+              <v-menu
+                :close-on-content-click="false"
+                offset-y
+                transition="slide-y-transition"
+              >
+                <template #activator="{ on }">
+                  <v-icon class="white--text" v-on="on"
+                    >fa-question-circle</v-icon
+                  >
+                </template>
+                <v-card tile class="pa-2 mt-1" max-width="320">
+                  <div class="body-2">
+                    All Prefect users are required to have a personal team for
+                    sandbox use. You can always switch between teams using the
+                    side menu. For more information about teams in Prefect,
+                    check out our
```

Comment:
```suggestion
                    Prefect automatically creates a sandbox development team for you to use - this is a great place to test and deploy flows as you explore Prefect. When you're ready to collaborate with others, you can <ExternalLink href="https://www.prefect.io/get-prefect#pricing">upgrade this team</ExternalLink> to invite more users or  <ExternalLink href="https://www.prefect.io/get-prefect/#contact">contact sales</ExternalLink> to add another team. For more information about teams in Prefect,
                    check out our
```
Let's change this wording to highlight what's happening here and the opportunities to upgrade. (you can accept this suggestion but the linting will fail)